frontstage/__init__.py

```diff
@@ -48,6 +49,10 @@
     frame_options="DENY",
 )
 redis = redis.Redis(host=app.config["REDIS_HOST"], port=app.config["REDIS_PORT"], db=app.config["REDIS_DB"])
+if app.config["TESTING"]:
```

Comment:
This should be written as:
```python
if app.config["TESTING"]:
    <make a fake redis>
else:
    <make a real one>
```

Also, can you put a comment in each path to say `logger.info('Using real redis')` and `logger.info('Using fake redis')` so there's less ways for a mistake to happen